### PR TITLE
Heroku

### PR DESCRIPTION
**Pull request by franciscoabalan**
## Summary
Create and configure Heroku remote server

## Trellocard
https://trello.com/c/m1lslVTK

## Screenshots
Captura de Pantalla 2019-10-14 a la(s) 12 49 20

Captura de Pantalla 2019-10-14 a la(s) 12 49 41

### CODE REVIEW

File: config/database.yml
Comment (joredudiaz) [inline config/database.yml]: use env variables for this data

Comment (franciscoabalan) [reply config/database.yml]: DONE

**Review by joredudiaz — APPROVED** [review]
let's wait for another approval 👍

**Review by aleochoam — APPROVED** [review]

**Pull request merged**
